marklogic-data-hub-central/ui/src/components/flows/flows.tsx

```diff
@@ -323,7 +331,15 @@ const Flows: React.FC<Props> = (props) => {
             onCancel={() => onCancel()}
             width={350}
         >
-            <div style={{fontSize: '16px', padding: '10px'}}>Are you sure you want to add step "{stepName}" to flow "{flowName}"?</div>
+            <div style={{fontSize: '16px', padding: '10px'}}>
+                {
+                    isStepInFlow(stepName, flowName)
+                    ?
+                    <p>The step "{stepName}" is already in the flow "{flowName}". Would you like to add another instance?</p>
```

Comment:
@MLjyang, let's make the styling here consistent with what's in the other confirmation boxes (the styling was updated recently). Use bold styling instead of quotes:

<img width="418" alt="DHFPROD-6174_load-card-add-step" src="https://user-images.githubusercontent.com/477757/99285657-b22d2400-27ec-11eb-8fc1-f6f5a20f4226.png">

Reply:
Fixed.  Added changes to unit tests for this update